src/plopp/widgets/__init__.py

```diff
@@ -7,7 +7,7 @@
 from .cut3d import Cut3dTool, TriCutTool
 from .drawing import DrawingTool, PointsTool
 from .linesave import LineSaveTool
-from .slice import SliceWidget, slice_dims
+from .slice import RangeSliceWidget, SliceWidget, slice_dims
 from .toolbar import Toolbar, make_toolbar_canvas2d, make_toolbar_canvas3d
 from .tools import ButtonTool, ColorTool, ToggleTool
 
```

Comment:
@MridulS Can you fix? The CI is failing because we now need to add `RangeSliceWidget` to the `__all__` below.